cmd/firewall.go

```diff
@@ -186,8 +241,28 @@ func (c *firewallCmd) createRequestFromCLI() (*models.V1FirewallCreateRequest, e
 		Tags:               mcr.Tags,
 		Networks:           mcr.Networks,
 		Ips:                mcr.Ips,
+		FirewallRules:      firewallRules,
 	}, nil
 }
+func parseFirewallRulesFile() (*models.V1FirewallRules, error) {
+	if !viper.IsSet("firewall-rules-file") {
+		return nil, nil
+	}
+
+	firewallRulesFile := viper.GetString("firewall-rules-file")
+	if firewallRulesFile == "" {
+		return nil, nil
+	}
+
+	firewallRules, err := os.ReadFile(firewallRulesFile)
+	if err != nil {
+		return nil, err
+	}
+
+	var fwrules models.V1FirewallRules
+	err = yaml.Unmarshal([]byte(firewallRules), &fwrules)
```

Comment:
```suggestion
	err = yaml.Unmarshal(firewallRules, &fwrules)
```
firewallRules is already of type []byte

Reply:
Thanks, done.